src/arrayMethodUnshift.js

```diff
@@ -4,8 +4,14 @@
  * Implement method unshift
  */
 function applyCustomUnshift() {
-  [].__proto__.unshift2 = function(...elements) {
-    // write code here
+  [].__proto__.unshift2 = function(...arg) {
+    const newArray = [...arg, ...this];
```

Comment:
try implementing this method without creating an extra array. It's not needed here.